src/sentry/api/bases/organization_events.py

```diff
@@ -338,6 +338,8 @@ def get_unit_and_type(self, field: str, field_type: str) -> tuple[str | None, st
                 return "1/second", field_type
             elif field in ["epm()", "spm()", "tpm()", "sample_epm()"]:
                 return "1/minute", field_type
+            elif field.startswith("rate(") and field.endswith(")"):
+                return "1/second", field_type
```

Comment:
Not entirely sure how a string like `1/second` is even used, or how we might change it for rate.